### PR TITLE
e2e: don't remove member used to connect to etcd cluster

### LINKED ISSUE #7204: test: TestCtlV3MemberRemove
```
--- FAIL: TestCtlV3MemberRemove (4.29s)
	ctl_v3_member_test.go:91: read /dev/ptmx: input/output error (expected "9366cce900fbd794 removed from cluster 4529f4d7b68cdaf9", got ["Error:  etcdserver: server stopped\r\n"])
```

### PR DESCRIPTION
**Pull request by heyitsanthony**
Fixes #7204

### CODE REVIEW

File: e2e/ctl_v3_member_test.go [hunk 1]
```diff
@@ -83,10 +83,7 @@ func memberRemoveTest(cx ctlCtx) {
 		cx.t.Fatalf("expected %d, got %d", n1, len(resp.Members))
 	}
```
Comment (fanminshi) [inline e2e/ctl_v3_member_test.go]: The checks performed by line 74 to 84 is already done by `memberToRemove()`. Remove those?

Comment (heyitsanthony) [inline e2e/ctl_v3_member_test.go]: yeah that was the intent, oops

**Comment by fanminshi** [comment]
lgtm defer to @xiang90

**Comment by xiang90** [comment]
LGTM

**Pull request merged**